arangod/RocksDBEngine/ReplicatedRocksDBTransactionCollection.cpp

```diff
@@ -227,24 +221,27 @@ auto ReplicatedRocksDBTransactionCollection::ensureCollection() -> Result {
 futures::Future<Result>
 ReplicatedRocksDBTransactionCollection::performIntermediateCommitIfRequired() {
   if (_rocksMethods->isIntermediateCommitNeeded()) {
+    // TODO due to distributeShardsLike, it is possible that we'll replicate
```

Comment:
Do we have a ticket for this?

Reply:
Created one: [CINFRA-698](https://arangodb.atlassian.net/browse/CINFRA-698)